tests/test_flow.py

```diff
@@ -3324,3 +3328,68 @@ def test_rename():
     controller.terminate()
     termination_result = controller.await_termination()
     assert termination_result == [{'b': 1, 'c': 3, 'e': 5}]
+
+
+def test_redis_driver_write(redis):
```

Comment:
This file is for unit tests. These are actually integration tests when `REDIS_URL` is set.

Reply:
well, we do have V3io driver running on other tests in this file - what's the difference?
anyway, can move it to integration tests if you prefer.

Reply:
There is no V3IO driver running in `test_flow.py`, because it's a unit test suite. Ideally fake redis will run in unit tests and real redis in integration tests, and without code duplication of course :).

Reply:
moved redis specific tests to out of test_flow.py and into new integration file test_redis_specific.py